gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -354,3 +366,45 @@ ds:
 liquibase:
   enabled: false
 
+alerts:
+  risk_assessment:
+    settings:
+      ipReputation:
+        enabled: true
+        thresholds:
+          - LOW: 1.0
+          - MEDIUM: 30.0
+          - HIGH: 70.0
+      geoVelocity:
+        enabled: true
+        thresholds:
+          - LOW: (5.0 / 18.0)
+          - MEDIUM: (125.0 / 18)
+          - HIGH: (625.0 / 9.0)
+      device:
+        enabled: true
+        thresholds:
+          - LOW: 1.0
+          - MEDIUM: 1.0
+          - HIGH: 1.0
+    severity: INFO
+  alert-engine:
+    enabled: true
+    ws:
+      discovery: true
+      endpoints:
+        - http://localhost:8072/
+      security:
+        username: admin
+        password: adminadmin
+      too_many_login_failures:
+        name: "Too many login failures detected"
+        description: "More than {threshold}% of logins are in failure over the last {window} second(s)"
+        # the threshold rate in % to reach before notify. Default 10% of login failures.
+        threshold: 1
+        # the minimum sample size. Default 1000 login attempts.
+        sampleSize: 1
+        # window time in seconds. Default 600s (10 minutes).
+        window: 10
+        # severity of the alert (INFO, WARNING, CRITICAL). Default WARNING.
+        severity: WARNING
```

Comment:
We may have to document these but we should not have them directly in the `gravitee.yml`

Reply:
Which ones?

Reply:
Ah I understand.